### DISCUSSION TITLE
[Feature] Adding a native way to add this to Home Assistant

### DISCUSSION THREAD

**Discussion by chris-burrow-apps** [damongolding/immich-kiosk#218] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Right now there seems to be no easy way to display this as a dashboard on Google Home Hubs. 
In the mean time, I have created my own tutorial on how you can do this with a blueprint which I will update when I think of new features 👍 

https://github.com/chris-burrow-apps/home-assistant-blueprints/blob/main/immich/immich_kiosk_mode.md

**Comment by damongolding** [comment]
Nice! Happy to accept a PR if you want to add a link in the Home Assistant section of the ReadMe 😊

**Reply by chris-burrow-apps** [reply]
Thanks 😄. I will work on the PR tomorrow.

**Reply by chris-burrow-apps** [reply]
I see you have merged my PR. Thanks a lot for doing that 👍
